### PR TITLE
add FAQ Section for multipleOf

### LINKED ISSUE #221: Add a FAQ question for 'multipleOf'
Explain how the "problematic" cases of multipleOf can be handled. See https://github.com/eclipsesource/jsonforms/issues/1897 for more information.

### PR DESCRIPTION
**Pull request by LukasBoll**
closes #221

Signed-off-by: Lukas Boll lukas-bool@web.de

### CODE REVIEW

File: content/faq/faq.mdx [hunk 1]
```diff
@@ -145,3 +145,40 @@ Now default values within the schema file can be used:
   "default": "Max"
 }
 ```
+## How can I use multipleOf with values smaller 1?
+
+JSON Forms uses AJV for the validation of data. By default, the `multipleOf` option of AJV only works with integers. If you are using fractional dividers, you also need to set the `multipleOfPrecision` option in AJV and pass your AJV instance to JSON Forms. To create the AJV instance with suitable options, you can call the `createAjv(options)` method exported by JSON Forms Core.
```
Comment (lucas-koehler) [inline content/faq/faq.mdx]: Please use a new line for each sentence. This makes it easier to diff changes in the future :)

File: content/faq/faq.mdx [hunk 2]
```diff
@@ -145,3 +145,40 @@ Now default values within the schema file can be used:
...
+AJV suggests setting the `multipleOfPrecision` to some positive integer N using this formula:
+
+`Math.abs(Math.round(multipleOf) - multipleOf) < 1e-N`
+
+In the Following example `multipleOf` is set to 0.1 and `multipleOfPrecision` is set to 2.
```
Comment (lucas-koehler) [inline content/faq/faq.mdx]:
```suggestion
In the following example `multipleOf` is set to `0.1` and `multipleOfPrecision` is set to `2`.
```

File: content/faq/faq.mdx [hunk 3]
```diff
@@ -145,3 +145,40 @@ Now default values within the schema file can be used:
...
+
+```js
+const schema = {
+    …
```
Comment (lucas-koehler) [inline content/faq/faq.mdx]: I would remove this line

File: content/faq/faq.mdx [hunk 4]
```diff
@@ -145,3 +145,40 @@ Now default values within the schema file can be used:
...
+    price: {
+      type: 'number',
+      multipleOf: 0.1
+    },
+    …
```
Comment (lucas-koehler) [inline content/faq/faq.mdx]: I would remove this line

File: content/faq/faq.mdx [hunk 5]
```diff
@@ -145,3 +145,40 @@ Now default values within the schema file can be used:
...
+};
+```
+
+
+```js
+import { createAjv } from '@jsonforms/core';
+import Ajv from 'ajv';
+
+const [ajvInstance] = useState<Ajv>(createAjv({multipleOfPrecision:2}));
```
Comment (lucas-koehler) [inline content/faq/faq.mdx]:
```suggestion
const [ajvInstance] = useState<Ajv>(createAjv({ multipleOfPrecision: 2 }));
```

File: content/faq/faq.mdx [hunk 6]
```diff
@@ -145,3 +145,40 @@ Now default values within the schema file can be used:
   "default": "Max"
 }
 ```
+## How can I use multipleOf with values smaller 1?
```
Comment (lucas-koehler) [inline content/faq/faq.mdx]:
```suggestion
## How can I use multipleOf with values smaller than 1?
```

**Review by lucas-koehler — CHANGES_REQUESTED** [review]
Thanks for the documentation. Looks already pretty good to me execpt for some minor issues (see inline comments) :)

**Comment by LukasBoll** [comment]
Thanks for the review, I updated the pull request according to your suggestions.

**Review by lucas-koehler — APPROVED** [review]
Thanks for the updates. LGTM now!

**Pull request merged**